README.rst

```diff
@@ -135,7 +135,7 @@ It is possible to make tests with expected error output, for example, if
 adding a new message or simply accepting that pylint is supposed to warn.
 A ``test_file_name.txt`` file contains a list of expected error messages in the
 format
-``error-type:line number:class name or empty:1st line of detailed error text``.
+``error-type:line number:class name or empty:1st line of detailed error text:confidence or empty``.
```

Comment:
Is this why the rest of the test suite is failing ? b/c we're missing the confidence value in expected results ?

Reply:
Unfortunately not. The tests are failing because the Pylint package doesn't include its test package in releases anymore. Pylint uses the `test_functional.py` module of Pylints test package.

I opened issue #250 for this problem